src/couch_mrview_util.erl

```diff
@@ -212,8 +212,11 @@ init_state(Db, Fd, State, #index_header{
     purge_seq=PurgeSeq,
     id_btree_state=IdBtreeState,
     view_states=ViewStates}) ->
-    NewViewStates = lists:map(fun({Pos, {Cnts, UsrReds, _DtSize}, Size}) ->
-        {{Pos, {Cnts, UsrReds}, Size}, nil, nil, Seq, PurgeSeq}
+    NewViewStates = lists:map(fun
+        (nil) ->
```

Comment:
@iilyak That's what I thought, naturally, but I assure you we would and that is what this fix is all about. As counter intuitive as it is, it is possible to get index with btree in `id_btree_state` in #index_header, but with `nil` in _one_ of the `view_states`, I can reproduce it quite reliably. All it takes is for one of the view to never emit.